javatests/arcs/showcase/inline/Reader.kt

```diff
@@ -3,16 +3,16 @@
 package arcs.showcase.inline
 
 import arcs.jvm.host.TargetHost
+import arcs.sdk.withParticleContext
 import arcs.showcase.ShowcaseHost
 import kotlinx.coroutines.ExperimentalCoroutinesApi
-import kotlinx.coroutines.withContext
 
 @ExperimentalCoroutinesApi
 @TargetHost(ShowcaseHost::class)
 class Reader0 : AbstractReader0() {
     private fun Level0.fromArcs() = MyLevel0(name)
 
-    suspend fun read(): List<MyLevel0> = withContext(handles.level0.dispatcher) {
+    suspend fun read(): List<MyLevel0> = withParticleContext {
```

Comment:
This is the kind of pattern we typically want to discourage (except in very distinct ingress & egress particles). Adding `withParticleContext` makes it even easier to continue the [typically-]antipattern of treating particles in this way.

Reply:
+1, we don't want it to be (or seem) easy to interact with particles like this.